python/frontend/cirrus/cirrus/messenger.py

```diff
@@ -27,8 +27,8 @@ def get_last_time_error(ip="127.0.0.1", port=1338):
         clientsocket = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
         clientsocket.sendto(GET_LAST_TIME_ERROR, (ip, port))
         clientsocket.settimeout(10)
-        s = clientsocket.recv(192)
-        return struct.unpack("ddd", s)
+        s = clientsocket.recv(256)
```

Comment:
This 256 probably should have a comment

Reply:
Num of bits of 4 doubles. Comment added.